published/009-2016-March-28.md

```diff
@@ -0,0 +1,239 @@
+
+### Orbit
+
+@haadcode has been working on improvements to [orbit-db](https://github.com/haadcode/orbit-db), [ipfs-log](https://github.com/haadcode/ipfs-log/) and [Orbit](https://github.com/haadcode/orbit). The message history fetching is now more stable and the UI feedback for loading messages is fixed. All this work will improve the user experience of Orbit.
+
+### js-ipfs-init
+
+`js-ipfs init` works! @noffle finished the remaining pieces this week, including CLI usage. This included a handful of [auxiliary](https://github.com/ipfs/js-ipfs/issue/91) [PRs](https://github.com/ipfs/js-ipfs/issue/96) that cascaded out of that work. This makes the `js-ipfs` init process produce IPFS repos that are compatible with go-ipfs' - this is big!
+
+#### Dictionary support for the zlib JavaScript Implementation, [`pako`](https://github.com/nodeca/pako)
+
+One of the significant contributions made this week was the addition of 'dictionary' support for `zlib` JavaScript implementation, `pako`. With this contribution, we are able to have a complete implementation of SPDY 3.1's framing layer running in the browser, the default stream muxing library used in IPFS. You can find more about this contribution in the following issue and PR discussions:
+
+- https://github.com/nodeca/pako/issues/69
+- https://github.com/diasdavid/js-libp2p-spdy/pull/6
+- https://github.com/nodeca/pako/pull/77
+
+### go-ipfs
+- @whyrusleeping wrote a tool to move content from 0.4.0 to 0.3.11 (https://github.com/whyrusleeping/levart-emit). He also discovered a file descriptor leak bug in utp causing connectivity issues, and began work on datastore performance improvements.
+
+#### `jsipfs object` cli and http-api endpoints are complete
+
+Now you can use `jsipfs object` in the same way you would use `ipfs object`. Big thanks to [Francisco Dias](https://github.com/xicombd) for leading the last miles of this goal. The complete track of the development can be found at [github.com/ipfs/js-ipfs/issues/58](https://github.com/ipfs/js-ipfs/issues/58).
+
+### Nginx metrics
+
+Our internal metrics dashboard didn't previously have HTTP request/response metrics from nginx's point of view, but only from IPFS's and [multireq](https://github.com/whyrusleeping/multireq)'s point of view. (Multireq is our v04x/v03x multiplexing proxy). Nginx itself provides finegrained metrics only through their commercial subscriptions. We're now using [mtail](https://github.com/google/mtail) to parse metrics from nginx access logs and expose them to [Prometheus](https://prometheus.io). @lgierth will also contribute the `nginx.mtail` program upstream with mtail.
+
+## Community
+
+### Upcoming talks
+
+On April 20th, IPFS will host a joint meetup with ConsenSys at MIT in Cambridge, Massachusetts. [Sign up here!](https://www.eventbrite.com/e/ipfs-ethereum-mit-meetup-tickets-24578852008)
+
+### First IPFS Meeting in NYC
```

Comment:
actually, just remove it -- it's a huge part of this weekly, which is already very big. I think it would be fine to just mention it happened here, and say another upcoming post detailing it will be published shortly.